### PR TITLE
Supported new mode: SCREEN_ANIMATION

### PR DESCRIPTION
**Pull request by mayank1513**
Hi,

I realized that screen could only be captured as video and not as animated gif. Thus, I added a new mode. No changes in the options object.

Earlier, if options object contained `animated: true`, `screen` option would be ignored. Now, by default screen is recorded as video but one can override this by adding animation options to turn it into `gif`. 

Hope this helps.

### CODE REVIEW

File: src/js/engine/record-mode.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /**
  * @file record-mode.js
- * @since 2.0.0
+ * @since 4.1.1
```
Comment (thijstriemstra) [inline src/js/engine/record-mode.js:3]: no need to change this.

**Review by thijstriemstra — CHANGES_REQUESTED** [review]
thanks! can you revert the whitespace changes in this pull request? and add an entry to the changelog?

File: package-lock.json [hunk 2]
```diff
@@ -2592,9 +2592,9 @@
       "dev": true
```
Comment (thijstriemstra) [inline package-lock.json:2592]: can you revert the changes to package-lock.json as well

**Comment by mayank1513** [comment]
Hi,

I will do it as soon as possible. and also respond to your other email
regarding the issue as soon as possible. Little busy today.

Mayank Kumar Chaudhari

**Comment by mayank1513** [comment]
Hi, sorry for delay. I am not able to create a new pull request now.

**Comment by thijstriemstra** [comment]
thanks for the pull request @mayank1513. While it's a good suggestion, I'd rather not expand on the old, buggy gif-recorder currently used in videojs-record to create animated GIFs. I started a new implementation, using gifshot (#292), but ran into other issues and the fact that that library is also not maintained anymore. Eventually we'll find a better implementation but I'm hesitant to build new things on top of the gif-recorder... thoughts?

**Pull request closed without merging**